patches/tModLoader/Terraria/ModLoader/UI/UIExtractMod.cs

```diff
@@ -94,7 +94,7 @@ internal class UIExtractMod : UIProgress
 					}
 
 					// Copy the dll to ModLoader\references\mods for easy collaboration.
-					if (name == "All.dll" || PlatformUtilities.IsXNA ? name == "Windows.dll" || name == $"{mod.Name}.XNA.dll" : name == "Mono.dll" || name == $"{mod.Name}.FNA.dll") {
+					if (name == "All.dll" || PlatformUtilities.IsXNA ? name == "Windows.dll" || name == $"{mod.Name}.XNA.dll" : name == "Mono.dll" || name == $"{mod.Name}.FNA.dll" || name == $"{mod.Name}.dll") {
```

Comment:
Can't we get rid of the `.FNA.dll` check now, too, since they're always FNA?

Reply:
Oh, yeah, we can also get rid of the legacy `.dll` checks (`Windows.dll` and `Mono.dll`), as well as `.XNA.dll` files. I *believe* they've all been streamlined to `ModName.dll`.